src/components/ArticleList.jsx

```diff
@@ -1,70 +1,107 @@
 import { useState, useEffect } from "react";
-import getArticleList from"../../websiteAPI";
+import getArticleList from "../../websiteAPI";
 import { Link } from "react-router-dom";
 
 const ArticleList = () => {
-    const [allArticles, setAllArticles] = useState([])
-    const [isLoading, setLoading] = useState(false)
-    const [isError, setError] = useState(false)
+  const [allArticles, setAllArticles] = useState([]);
+  const [viewedArticle, setViewedArticle] = useState(null);
+  const [isLoading, setLoading] = useState(false);
+  const [isError, setError] = useState(false);
 
-    useEffect(() => {
-        setLoading(true)
-        setError(false)
-    getArticleList().then((response) => {
-        setLoading(true)
-        const articles = response.data
-        setAllArticles(articles)
-    })
-    .catch((err) => {
-        setLoading(false)
-        setError(err)
-    })
-    },[])
-    if(isLoading) {
-        return <>
+  useEffect(() => {
+    setLoading(true);
+    setError(false);
+    getArticleList()
+      .then((response) => {
+        setLoading(false);
+        const articles = response.data;
+        setAllArticles(articles);
+      })
+      .catch((err) => {
+        setLoading(false);
+        setError(err);
+      });
+  }, []);
+
+  function onClick(viewedArticle) {
+    setViewedArticle(viewedArticle);
+  }
+
+  if (isLoading) {
+    return (
+      <>
         <img className="Loading" src="src/assets/loading-wheel.svg"></img>
-        <p className='normalText'>Hang tight</p>
-        <p className='normalText'>Things are happening...</p>
-        </>
-    }
+        <p className="normalText">Hang tight</p>
+        <p className="normalText">Things are happening...</p>
+      </>
+    );
+  }
 
-    if(isError) {
-        console.log(isError)
-        return <>
+  if (isError) {
+    console.log(isError);
+    return (
+      <>
         <img className="errorImage" src=""></img>
-        <p className='normalText'>Status: {isError.status}....Uh Oh</p>
-        </>
-    }
+        <p className="normalText">Status: {isError.status}....Uh Oh</p>
+      </>
+    );
+  }
+  if(viewedArticle){
     return (
-        <section className="ArticleList">
-        <h1>Articles</h1>
-        <nav>
-          <Link className="link" to="/">
-            Home
-          </Link>
-        </nav>
-        <section className='Map'>
+        <section className="JustArticle">
+                      <nav>
+        <Link className="link" to="/">
+          Home
+        </Link>
+      </nav>
+          <h2> {viewedArticle.props.children[1].props.children[1]}</h2>
+          {viewedArticle}
+        </section>
+      );
+  }
```

Comment:
This should not be rendered like this and should exist within its own component page based on the url on the frontend side. This would also help with the way this h2 is being displayed.